snappy/dpu_snappy.c

```diff
@@ -6,11 +6,22 @@
 #include <getopt.h>
 #include <time.h>
 
+
 #include "dpu_snappy.h"
 
 #define DPU_DECOMPRESS_PROGRAM "dpu-decompress/decompress.dpu"
 #define TOTAL_NR_TASKLETS (NR_DPUS * NR_TASKLETS)
 
+/* This should be coming from the config but the include isn't working for me. */
+/* #include <perfcounter.h> */
```

Comment:
I didn't check, but I believe <perfcounter.h> is a "dpu only" header, and this file is compiled for the host.